stdlib/ballerina-websub/src/main/ballerina/websub/subscriber_service_endpoint.bal

```diff
@@ -281,8 +291,8 @@ returns (http:FilterResult) {
 @Description {value:"Function to invoke the WebSubSubscriberConnector's actions for subscription"}
 @Param {value:"hub: The hub to which the subscription request is to be sent"}
 @Param {value:"subscriptionDetails: Map containing subscription details"}
-function invokeClientConnectorForSubscription (string hub, map subscriptionDetails) {
-    endpoint Client websubHubClientEP { url:hub };
+function invokeClientConnectorForSubscription (string hub, http:AuthConfig | () auth, map subscriptionDetails) {
```

Comment:
Shall we consistently use `?` over `()`?

Reply:
fixed with 9c803e8